app/components/external-link.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: "a",
+  href: "#",
+
+  click: function() {
+    window.open(this.decodeLink(), "_system");
```

Comment:
@steveyken thanks for taking a look, I have not noticed that it will be removed in future release. 
As it was a common component for both web and cordova, I added window.open.
We can add condition and use  "cordova.InAppBrowser.open"  in cordova app only.